libs/wingsdk/src/std/test.ts

```diff
@@ -43,8 +43,18 @@ export class Test extends Resource {
     Node.of(this).title = "Test";
     Node.of(this).description = "A cloud unit test.";
 
-    if (App.of(this).isTestEnvironment || App.of(this)._target === "sim") {
```

Comment:
I wonder if this could be encapsulated:

```js
App.of(this)._testRunner?.addTestFunction(this, "Handler", inflight, props);
```

And basically keep track of everything inside the runner.

Reply:
Added